tools/cellprofiler/runcp.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+import sys
+import os
+from random import randint
+
+DEVNULL = open(os.devnull,'wb') #This is needed to supress ilastik error due to running CellProfiler in headless configuration
+cp_path = os.environ['CP_PATH']
+outpath = str(randint(1,100000))
+tmppath = cp_path + '/bcc/' + outpath 
```

Comment:
General rule of Python - there is a library for that :)
See tempfile library, in particular the mkdtemp() function.